src/utils.ts

```diff
@@ -127,42 +142,54 @@ export async function buildIpfs(configObj): Promise<any> {
 }
 
 export async function buildCeramic(configObj, ipfs?: IpfsApi): Promise<CeramicApi> {
-  let ceramic
   if (configObj.mode == 'client') {
     console.log(`Creating ceramic via http client, connected to ${configObj.apiURL}`)
-    ceramic = new CeramicClient(configObj.apiURL, { syncInterval: 500 })
+
+    const ceramic = new CeramicClient(configObj.apiURL, { syncInterval: 500 })
+    const did = await createDid(seed)
+    ceramic.did = did
+
     console.log(`Ceramic client connected successfully to ${configObj.apiURL}`)
+    return ceramic
   } else if (configObj.mode == 'node') {
     console.log('Creating ceramic local node')
+
+    process.env.CERAMIC_ENABLE_EXPERIMENTAL_COMPOSE_DB = 'true'
     const loggerProvider = new LoggerProvider({ logLevel: LogLevel.debug })
+    const indexingDirectory = await tmp.dir({ unsafeCleanup: true })
+
     const ceramicConfig: CeramicConfig = {
       networkName: configObj.network,
       ethereumRpcUrl: configObj.ethereumRpc,
       anchorServiceUrl: configObj.anchorServiceAPI,
-      loggerProvider
+      loggerProvider,
+      indexing: {
+        db: `sqlite://${indexingDirectory.path}/ceramic.sqlite`,
+        models: [Model.MODEL],
+        allowQueriesBeforeHistoricalSync: true
+      }
     }
     const [modules, params] = await Ceramic._processConfig(ipfs, ceramicConfig)
     if (configObj.s3StateStoreBucketName) {
       const bucketName = `${configObj.s3StateStoreBucketName}${S3_DIRECTORY_NAME}`
       const s3StateStore = new S3StateStore(bucketName)
       modules.pinStoreFactory.setStateStore(s3StateStore)
     }
-    ceramic = new Ceramic(modules, params)
-    await ceramic._init(true, true)
+
+    const ceramic = new Ceramic(modules, params)
+    await ceramic._init(true)
+    const did = await createDid(seed)
```

Comment:
when you update to the newest ceramic version you'll need to make sure that ceramic.did is set before calling `ceramic._init()`.  This is because part of calling `ceramic._init()` now involves loading the Models that are configured to be indexed via the config file (so we can see if they have any relations configured and create the appropriate columns in the indexing database if so), which requires having a DID resolver to use.